app/cloud_tasks/cloud_task_publishers.py

```diff
@@ -54,5 +51,5 @@ def create_task(self, payload: bytes, queue_name: str) -> None:
 
 class LogCloudTaskPublisher:
     @staticmethod
-    def create_task(payload: bytes, queue_name: str) -> None:
-        logger.info("creating cloud task", payload=payload, queue_name=queue_name)
+    def create_task(task: dict, queue_name: str) -> None:
```

Comment:
don't use `dict` for type hints where possible - the typing library provides `Mapping` or `MutableMapping`